### DISCUSSION TITLE
Makegood MG-AUWF01 - How to flip switch ordering in UI?

### DISCUSSION THREAD

**Discussion by victorhooi** [arendst/Tasmota#12170] [Q&A] [answered]
I bought a [Makegood MG-AUWF01](https://templates.blakadder.com/makegood_MG-AUWF01.html), and was able to flash it to Tasmota using tuya-convert.

The Tasmota web UI toggles are the reverse ordering of the physical sockets on the device.

I'm guessing this is due to the template I applied from the wiki?

```
{"NAME":"MG-AUWF01","GPIO":[56,10,157,59,134,132,0,0,131,22,57,21,9],"FLAG":0,"BASE":18}
````

Is there a way to flip the ordering from here?

Also - if I want to make changes to the device template itself, - what's the best way to do that? Do I just make a PR from the page, or is there some kind of process/discussion that needs to happen?

**Comment by sfromis** [comment, marked as answer]
You can move the pins numbers around using Configure Template. There's not much of a formal process for changes to the templates repository, many changes are simply someone spotting an issue and making a PR, stating their case.
